lib/signaling/v2/twilioconnectiontransport.js

```diff
@@ -485,7 +497,6 @@ function setupEventListeners(transport) {
     const twilioConnection = createOrResetTwilioConnection();
     twilioConnection.once('close', disconnect);
     twilioConnection.on('message', handleMessage);
-    twilioConnection.once('open', connect);
```

Comment:
since we do not use open event anymore, how about stopping twilio connection from emitting it? - I am not very sure about this though - given that we have this logger approroach to events now.